lib/Mojo/Server/Daemon.pm

```diff
@@ -194,7 +194,8 @@ sub _listen {
       my ($loop, $stream, $id) = @_;
 
       $self->{connections}{$id} = {tls => $tls};
-      warn "-- Accept $id (@{[$stream->handle->peerhost]})\n" if DEBUG;
+      my $peer = $stream->handle->isa('IO::Socket::UNIX') ? 'peerpath' : 'peeraddr' if DEBUG;
```

Comment:
Shouldn't this still be using peerhost rather than peeraddr? peeraddr according to the documentation is a binary octet string which would not be suitable for printing

Reply:
Stupid of me!  I didn't meant to change it from `peerhost` to `peeraddr`.  I'll change that when I improve the rest of the patch.  Thanks for catching that.